types/stripe-v3/index.d.ts

```diff
@@ -44,27 +44,47 @@ declare namespace stripe {
         retrievePaymentIntent(
             clientSecret: string,
         ): Promise<PaymentIntentResponse>;
+        /** @deprecated */
         handleCardPayment(
             clientSecret: string,
             element: elements.Element,
             options?: HandleCardPaymentOptions,
         ): Promise<PaymentIntentResponse>;
+        /** @deprecated */
         handleCardPayment(
             clientSecret: string,
             options?: HandleCardPaymentWithoutElementsOptions,
         ): Promise<PaymentIntentResponse>;
+        confirmCardPayment(
+            clientSecret: string,
+            data?: ConfirmCardPaymentOptions,
+        ): Promise<PaymentIntentResponse>;
         handleCardAction(
             clientSecret: string,
         ): Promise<PaymentIntentResponse>;
+        confirmSepaDebitPayment(
+            clientSecret: string,
+            data?: ConfirmSepaDebitPaymentOptions,
+        ): Promise<PaymentIntentResponse>;
+        /** @deprecated */
         handleCardSetup(
             clientSecret: string,
             element: elements.Element,
             data?: HandleCardSetupOptions,
         ): Promise<SetupIntentResponse>;
+        /** @deprecated */
         handleCardSetup(
             clientSecret: string,
             data?: HandleCardSetupOptionsWithoutElementsOptions,
         ): Promise<SetupIntentResponse>;
+        confirmCardSetup(
+            clientSecret: string,
+            data?: ConfirmCardSetupOptions,
+        ): Promise<SetupIntentResponse>;
```

Comment:
Perhaps the [argument `options`](https://stripe.com/docs/js/setup_intents/confirm_card_setup#stripe_confirm_card_setup-options) was missed?